### ISSUE TITLE
Support LDF endpoint?

### ISSUE THREAD

**Issue by dpriskorn** [LeMyst/WikibaseIntegrator#195]
With this we could encourage people writing queries to offload to LDF when possible and do more computing locally now that WDQS expensive SPARQL endpoint receives >100.000 queries a second.

see https://github.com/pchampin/hydra-py for a Triple Patterns Fragments client

**Comment by LeMyst** [comment]
I never used LDF or TPF.
That seems interesting and, if I understand correctly, that can solve the timeout when querying the SPARQL server.

I'll not put this at the top of the list, for the moment, but that seems promising.

**Comment by dpriskorn** [comment]
Exactly, no timeouts but often a truck load of requests because we get only 100 results at a time.
Complex queries take more time.
